serving/src/main/java/ai/djl/serving/http/InferenceRequestHandler.java

```diff
@@ -131,14 +133,23 @@ protected void handleRequest(
                                 });
                 break;
             case "invocations":
-                handleInvocations(ctx, req, decoder, null);
+                if (!Boolean.parseBoolean(Utils.getenv("MESSAGES_API_ENABLED"))) {
```

Comment:
this shouldn't be `!`, right? if the environment variable is true we should handle as chat completion?